llvm/test/CodeGen/DirectX/length.ll

```diff
@@ -0,0 +1,131 @@
+; RUN: opt -S  -dxil-intrinsic-expansion  < %s | FileCheck %s --check-prefixes=CHECK,EXPCHECK
+; RUN: opt -S  -dxil-op-lower -mtriple=dxil-pc-shadermodel6.3-library < %s | FileCheck %s --check-prefixes=CHECK,DOPCHECK
+
+declare half @llvm.fabs.f16(half)
```

Comment:
We ussually add a comment at the top describing the test case. 
Like for clamp we said:
`; Make sure dxil operation function calls for clamp/uclamp are generated for half/float/double/i16/i32/i64.`